# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/canvas/OffscreenCanvas.cpp
# Commit: 5ad078db7d7b
# Full Hash: 5ad078db7d7b025ae400922fde028352a430592d
# Author: Andrew Osmond <aosmond@mozilla.com>
# Date: 2021-12-10 21:58:52
# Regressor Bug: 1736177
# File Overlap Count: 2
# Description:
#   Bug 1736177 - Part 1. Rename CanvasRenderingContextHelper::GetContext to GetOrCreateContext. r=jgilbert
#   
#   This patch contains no functional changes but sets us up for refactoring
#   in a later part in this series. It also helps silence a bunch of
#   compiler warnings.
# ==============================================================================

diff -r e59ba9934ece -r 5ad078db7d7b dom/canvas/OffscreenCanvas.cpp
--- a/dom/canvas/OffscreenCanvas.cpp	Fri Dec 10 02:57:51 2021 +0000
+++ b/dom/canvas/OffscreenCanvas.cpp	Fri Dec 10 02:57:51 2021 +0000
@@ -111,8 +111,8 @@
     return nullptr;
   }
 
-  RefPtr<nsISupports> result = CanvasRenderingContextHelper::GetContext(
-      aCx, aContextId, aContextOptions, aRv);
+  RefPtr<nsISupports> result = CanvasRenderingContextHelper::GetOrCreateContext(
+      aCx, contextType, aContextOptions, aRv);
 
   if (!mCurrentContext) {
     return nullptr;